packages/cms-cli/commands/customObject/schema/fetch-all.js

```diff
@@ -49,8 +50,7 @@ exports.builder = yargs => {
   ]);
 
   yargs.positional('dest', {
-    describe:
-      'Local folder where schemas will be written.  If omitted, current working directory will be used',
+    describe: 'Local folder where schemas will be written',
```

Comment:
Decided to remove this.  I think a lot of CLIs use CWD if a path isn't specified, so hoping that it is implied.  Just trying to reduce the amount of info to end users